tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/Defaults.scala

```diff
@@ -26,50 +30,49 @@ sealed trait RunnerDefaults {
 final case class AkkaRunnerDefaults(
     resourceConstraints: ResourceConstraints,
     javaOptions: String,
-    prometheusRules: String
-) extends RunnerDefaults
+    prometheusRules: String)
+    extends RunnerDefaults
 
 final case class ResourceConstraints(
     cpuRequests: Quantity,
     memoryRequests: Quantity,
     cpuLimits: Option[Quantity],
-    memoryLimits: Option[Quantity]
-)
+    memoryLimits: Option[Quantity])
 
 final case class SparkPodDefaults(
     cores: Option[Quantity],
     memory: Option[Quantity],
     coreLimit: Option[Quantity],
     memoryOverhead: Option[Quantity],
-    javaOptions: Option[String]
-)
+    javaOptions: Option[String])
 
 final case class SparkRunnerDefaults(
     driverDefaults: SparkPodDefaults,
     executorDefaults: SparkPodDefaults,
-    prometheusRules: String
-) extends RunnerDefaults
+    prometheusRules: String)
+    extends RunnerDefaults
 
 final case class FlinkPodResourceDefaults(
     cpuRequest: Option[Quantity] = None,
     memoryRequest: Option[Quantity] = None,
     cpuLimit: Option[Quantity] = None,
-    memoryLimit: Option[Quantity] = None
-)
+    memoryLimit: Option[Quantity] = None)
 
-final case class FlinkJobManagerDefaults(
-    replicas: Int,
-    resources: FlinkPodResourceDefaults
-)
+final case class FlinkJobManagerDefaults(replicas: Int, resources: FlinkPodResourceDefaults)
 
-final case class FlinkTaskManagerDefaults(
-    taskSlots: Int,
-    resources: FlinkPodResourceDefaults
-)
+final case class FlinkTaskManagerDefaults(taskSlots: Int, resources: FlinkPodResourceDefaults)
 
 final case class FlinkRunnerDefaults(
     parallelism: Int,
     jobManagerDefaults: FlinkJobManagerDefaults,
     taskManagerDefaults: FlinkTaskManagerDefaults,
-    prometheusRules: String
-) extends RunnerDefaults
+    prometheusRules: String)
+    extends RunnerDefaults
+
+object Common {
```

Comment:
Change to `JsonConfig` object with apply that returns `Config`, or just copy paste that one line in the two places you use it.

Also does not really seem to fit in `Defaults` file...

Reply:
I like `JsonConfig`